### PR TITLE
rpmbuild: add conditions around ASE packaging

### PR DESCRIPTION
**Pull request by r-rojo**

### CODE REVIEW

**Comment by ibarraza** [comment]
This does not appear to allow ASE to build or package even if BUILD_ASE=ON?

**Comment by ibarraza** [comment]
With BUILD_ASE=OFF, the generated top level opae-<version>.rpm package still requires the opae-ase package, which of course was not built.

File: opae.spec.in [hunk 1]
```diff
@@ -18,7 +18,7 @@ Source0:        @CPACK_PACKAGE_NAME@.tar.gz
 %define _unpackaged_files_terminate_build 0
```
Comment (tswhison) [inline opae.spec.in]: So, does line 12 need to be wrapped in..
```bash
%if %with_ase == "ON"
```

**Comment by tswhison** [comment]
@ibarraza could you try with the latest commit?  It seemed to work for me.

**Pull request merged**